packages/frontend/src/components/TableConfigPanel/ConditionalFormatting.tsx

```diff
@@ -84,9 +90,8 @@ const ConditionalFormatting: FC = () => {
 
         setConfig(
             produce(config, (draft) => {
-                // TODO: this is intentional to support multiple rules in the future
                 draft.rules[0] = newRule;
-                // TODO: check if we can fix this problem in number input
+                // FIXME: check if we can fix this problem in number input
```

Comment:
what's the issue here ? do we need to fix it ?

Reply:
I am not sure but sometimes number input returns string. I will deal with that in the following PR or just break out as a separate task